swarm-derive/Cargo.toml

```
@@ -3,7 +3,7 @@ name = "libp2p-swarm-derive"
 edition = "2021"
 rust-version = { workspace = true }
 description = "Procedural macros of libp2p-swarm"
-version = "0.34.1"
+version = "0.35.1"
```

```suggestion
version = "0.34.2"
```

This should work too I think. We pin the version so there shouldn't be a need for a minor version bump.

---

:heavy_check_mark: